com.leinardi.gst.json

```diff
@@ -0,0 +1,157 @@
+{
+  "app-id": "com.leinardi.gst",
+  "command": "gst",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.34",
+  "sdk": "org.gnome.Sdk",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=x11",
+    "--device=dri",
+    "--share=network",
+    "--talk-name=org.freedesktop.Flatpak",
+    "--filesystem=/sys:ro",
+    "--env=PATH=/usr/bin:/bin:/usr/sbin:/sbin:/app/bin:/app/sbin"
+  ],
+  "modules": [
+    {
+      "name": "lm-sensors",
+      "buildsystem": "simple",
+      "build-commands": [
+        "sed -i -e 's/ -g / -s /' Makefile",
+        "make -j $FLATPAK_BUILDER_N_JOBS",
+        "find . -type f | perl -lne 'print if -B and -x' | xargs strip",
+        "make install BUILD_STATIC_LIB=0 DEBUG=0 PREFIX=/ DESTDIR=$FLATPAK_DEST"
+      ],
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/lm-sensors/lm-sensors.git",
+          "tag": "V3-5-0",
+          "commit": "e8afbda10fba571c816abddcb5c8180afc435bba"
+        }
+      ]
+    },
+    {
+      "name": "libaio",
+      "config-opts": [
+        "--buildtype=debugoptimized"
+      ],
+      "buildsystem": "simple",
+      "build-commands": [
+        "make prefix=$FLATPAK_DEST install"
+      ],
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://pagure.io/libaio.git",
+          "tag": "libaio-0.3.111",
+          "commit": "f66be22ab0a59a39858900ab72a8c6a6e8b0b7ec"
+        }
+      ]
+    },
+    {
+      "name": "libbsd",
```

Comment:
libgcrypt is already in the freedesktop SDK.

Reply:
I just realized I swapped the modules names of `libbsd` and `libgcrypt`.

Just to avoid confusion, which one is already in the SDK? "https://www.gnupg.org/ftp/gcrypt/libgcrypt" or "https://gitlab.freedesktop.org/libbsd"?

Reply:
Removed `libgcrypt`, tested and everything seems to work as before.